File: ePrijevozSarajevo/ePrijevozSarajevo.Model/Manufacturer.cs
namespace ePrijevozSarajevo.Model
{
    public class Manufacturer
    {
        public int ManufacturerId { get; set; }
        public string? Name { get; set; }
        public DateTime? ModifiedDate { get; set; }
        public int ManufacturerCountryId { get; set; }

    }
}

File: ePrijevozSarajevo/UI/eprijevoz_desktop/lib/models/manufacturer.dart
import 'package:json_annotation/json_annotation.dart';

part 'manufacturer.g.dart';

@JsonSerializable()
class Manufacturer {
  int? manufacturerId;
  String? name;
  DateTime? modifiedDate;
  int? manufacturerCountryId;

  Manufacturer({this.manufacturerId, this.name});

  factory Manufacturer.fromJson(Map<String, dynamic> json) =>
      _$ManufacturerFromJson(json);

  /// Connect the generated [_$PersonToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$ManufacturerToJson(this);
}
